client/package.json

```
@@ -94,6 +94,8 @@
     "postbuild": "del-cli --force ../app/public && mv build ../app/public",
     "dev": "run-s start",
     "auto-test": "npm test -- --auto-watch --no-single-run",
-    "test": "karma start karma.config.js"
+    "test": "karma start karma.config.js",
+    "bpmn-io-modelers:test": "cross-env modelers=true npm test",
+    "bpmn-io-modelers:auto-test": "cross-env modelers=true npm run auto-test"
```

```suggestion
    "bpmn-io-modelers:test": "cross-env MODELERS=1 npm test",
    "bpmn-io-modelers:auto-test": "cross-env MODELERS=1 npm run auto-test"
```

Fixes https://github.com/camunda/camunda-modeler/pull/1811#issuecomment-643986034

The problem was caused by lowercased name in the env while using uppercased name in the code. I also changed the value so that there is no confusion when somebody changes it to `cross-env MODELERS=false` as a non-empty string is always truthy.

---

Ooops. Good catch! 👍